src/processing/script/index.js

```diff
@@ -117,19 +112,45 @@ export function isScriptProcessed (code) {
     return PROCESSED_SCRIPT_RE.test(code);
 }
 
-export function processScript (src, withHeader, beautify) {
+export function applyChanges (script, changes, isObject) {
+    var indexOffset = isObject ? -1 : 0;
+    var chunks      = [];
+    var index       = 0;
+
+    if (!changes.length)
+        return script;
+
+    for (var i = 0; i < changes.length; i++) {
+        var changeStart = changes[i].start + indexOffset;
+        var changeEnd   = changes[i].end + indexOffset;
+
+        chunks.push(script.substring(index, changeStart));
+        chunks.push(getCode(changes[i].replacement, script.substring(changeStart, changeEnd)));
+        index += changeEnd - index;
+    }
+
+    chunks.push(script.substring(index));
+
+    return chunks.join('');
+}
+
+export function processScript (src, withHeader) {
     var { bom, preprocessed } = preprocess(src);
-    var { ast, isObject }     = analyze(preprocessed);
+    var withoutHtmlComments = removeHtmlComments(preprocessed);
```

Comment:
It seems that the WebStorm is not able to properly format similar constructions :(